### PR TITLE
Address feedback

### PR DESCRIPTION
**Pull request by npm1**
Based on https://github.com/tdresser/element-timing/commit/56b1167157aee4b0f0610f8fd84f3d42ed223fad

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,20 +25,17 @@ TODO: fix implicit registration.
...
-We say that an element is **text-painted** if the following two conditions are satisfied:
-
-* At least one text node <em>belongs to</em> the element.
-* All of the text nodes which belong to the element have been painted at least once.
+We say that an element is **text-painted** if at least one text node <em>belongs to</em> and has been painted at least once.
 
 Thus, the **text rendering timestamp** of an element is the time when it becomes <em>text-painted</em>.
 
-Given a group of text nodes, we define the **text rect** as the geometric union of the text node display rectangles within the viewport.
+Given a group of text nodes, we define the **text rect** as the smallest rectangle which contains the geometric union of the text node display rectangles within the viewport.
```
Comment (tdresser) [inline README.md]: "the geometric union of" -> ""

File: README.md [hunk 2]
```diff
@@ -85,9 +82,9 @@ The entry creation might be affected by _visibility_: for instance, elements are
 
 #### What are some differences between text and image observation?
 
-* Whereas for images it is OK to care only about those which are associated to a resource, for text that is definitely not the case. An image not associated to a resource will need to be constructed manually and it is uncommon for such an image to be part of the key content of a website. On the other hand, text is relevant regardless of whether it is associated to a webfont or not. A webfont only makes it ‘pretty text’.
+* Whereas for images it is OK to care only about those which are associated to a resource, for text that is definitely not the case. An image not associated to a resource will need to be constructed manually and it is uncommon for such an image to be part of the key content of a website. Note that we consider inline images to have an associated resource, it is just inlined. On the other hand, text is relevant regardless of whether it is associated to a webfont or not.
```
Comment (tdresser) [inline README.md]: Is "inline" image a well defined concept?

File: README.md (hunk 1 above)
Comment (npm1) [reply README.md]: I don't understand this comment...

File: README.md (hunk 2 above)
Comment (npm1) [reply README.md]: I think it's well understood? I could add "(those with data URIs)"

Comment (tdresser) [reply README.md]: I think that would be a bit clearer.

**Pull request merged**